core-web/libs/contenttype-fields/src/lib/fields/binary-field/components/dot-ui-message/dot-ui-message.component.scss

```diff
@@ -0,0 +1,31 @@
+@use "variables" as *;
+
+:host {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    gap: $spacing-3;
+    height: 100%;
```

Comment:
Do we want this component to ALL the time be 100% height? or it will depend where is being used?